resources/test/dumps/Bacteria Hunt Speeds.js

```diff
@@ -165,7 +168,15 @@ var procedures = (function() {
           procedures["MAKE-FLAGELLA"]();
           SelfManager.self().setXY(Prims.randomCoord(world.topology.minPxcor, world.topology.maxPxcor), Prims.randomCoord(world.topology.minPycor, world.topology.maxPycor));
         }, true);
-      }), [1, 2, 3, 4, 5, 6]);
+      }, "[ this-variation ->
```

Comment:
Not sure whether there is planned work to do this, but I think it would be worthwhile to HTML-escape the special characters in this string.